fireapp/lib/exception/make_new_shift_exception.dart

```diff
@@ -0,0 +1,7 @@
+class MakeNewShiftException implements Exception {
```

Comment:
This describes what you're doing while making a shift, not what went wrong. It should describe what went wrong. Ditto for the other one